packages/@ourworldindata/grapher/src/stackedCharts/MarimekkoChart.tsx

```diff
@@ -959,6 +943,37 @@ export class MarimekkoChart
                 <DualAxisComponent dualAxis={dualAxis} showTickMarks={true} />
                 <HorizontalCategoricalColorLegend manager={this} />
                 {this.renderBars()}
+                {target && (
+                    <Tooltip
+                        id="marimekkoTooltip"
+                        tooltipManager={this.manager}
+                        x={position.x}
+                        y={position.y}
+                        style={{ maxWidth: "250px" }}
+                        offsetX={20}
+                        offsetY={-16}
+                        title={entityName}
+                        subtitle={entityColor?.colorDomainValue}
```

Comment:
Showing the year as a subtitle would make it more consistent with other chart types.